inspire_classifier/core/__init__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
Empty `__init__.py` files are not needed on python 3 to create a module: https://stackoverflow.com/questions/37139786/is-init-py-not-required-for-packages-in-python-3.